src/modules/LanItems.rb

```diff
@@ -645,7 +645,7 @@ def WriteUdevItemsRules
           # - remove it completely
           # removing is less error prone when tracking name changes, so it was chosen.
           item_udev_net = RemoveKeyFromUdevRule(item_udev_net, "KERNEL")
-          SetLinkDown(dev_name)
+          SetLinkDown(dev_name) if !Mode.autoinst
```

Comment:
I would put here also an comment, that AutoYaST would have here a problem if it used via SSH installation.